amundsen_application/static/js/components/DashboardPage/index.spec.tsx

```diff
@@ -14,38 +14,47 @@ describe('DashboardPage', () => {
     const routerProps = getMockRouterProps<RouteProps>({ uri: 'test:uri/value' }, null);
     const props = {
       isLoading: false,
+      statusCode: 200,
       dashboard: {
-        uri: 'test:uri/value',
-        cluster: 'cluster',
-        group_name: 'google group',
-        group_url: 'https://google.com',
-        name: 'lmgtfy dashboard',
-        url: 'https://lmgtfy.com/?q=dashboard',
-        description: 'test description',
-        created_timestamp: 1582153297,
-        updated_timestamp: 1582153297,
-        last_run_timestamp: 1582153297,
-        last_run_state: 'state',
-        owners: [{
-          email: 'dwon@lyft.com',
-          display_name: 'Daniel Won',
-          profile_url: "",
-          user_id: 'dwon@lyft.com',
-        }],
-        frequent_users: [{
-          read_count: 10,
-          user: {
-            email: 'dwon@lyft.com',
-            display_name: 'Daniel Won',
-            profile_url: "",
-            user_id: 'dwon@lyft.com',
+        badges: [],
```

Comment:
nit: creating the one time fixtures for the different chunks of data corresponding to at least the most important use cases to test would make tests across the app a bit less cumbersome to set up.